src/ui/base-page-storyEdit.js

```diff
@@ -18,6 +18,58 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
         pageSetup: {
             hiddenEditorClass: "hidden"
         },
+        model: {
+            // the initial state is only the editor ui showing.
+            // in much the same way as within the editor, the visibility
+            // of the editor and previewer are mutually exclusive.
+            // the individual steps of the editor (editStoryStep and metadataStep)
+            // are controlled within the editor model.
+            // in this way, the hiding and showing of the three steps of
+            // the Edit page can be achieved
+            editorVisible: true,
+            previewerVisible: false
+        },
+        modelRelay: {
+            editPageVisibilityMutex: {
+                source: "editorVisible",
+                target: "previewerVisible",
+                singleTransform: {
+                    type: "sjrk.storyTelling.transforms.not"
+                }
+            }
+        },
+        modelListeners: {
+            "editorVisible": [{
+                this: "{storyEditor}.container",
+                method: "toggle",
+                args: ["{change}.value"],
+                namespace: "manageEditorVisibility"
+            },
+            {
+                func: "{that}.events.onContextChangeRequested.fire",
```

Comment:
As the change of `model.editorVisible` automatically triggers the change of `model.previewerVisible` and vice versa, which means the change to any of these 2 model values will trigger `onContextChangeRequested` event to be fired twice. Is this correct?

Reply:
The double-firing is unintentional. I will remove the listener on `previewerVisible`